core/src/commonMain/kotlin/com/xebia/functional/xef/llm/Chat.kt

```diff
@@ -1,75 +1,80 @@
 package com.xebia.functional.xef.llm
 
+import com.xebia.functional.openai.apis.ChatApi
+import com.xebia.functional.openai.models.CreateChatCompletionRequest
+import com.xebia.functional.openai.models.CreateChatCompletionRequestModel
+import com.xebia.functional.openai.models.ext.chat.stream.createChatCompletionStream
 import com.xebia.functional.xef.AIError
 import com.xebia.functional.xef.conversation.AiDsl
 import com.xebia.functional.xef.conversation.Conversation
-import com.xebia.functional.xef.llm.models.chat.*
 import com.xebia.functional.xef.prompt.Prompt
 import com.xebia.functional.xef.prompt.templates.assistant
 import kotlinx.coroutines.flow.*
 
-interface Chat : LLM {
+@AiDsl
+fun ChatApi.promptStreaming(
+  prompt: Prompt<CreateChatCompletionRequestModel>,
+  scope: Conversation
+): Flow<String> = flow {
+  val messagesForRequestPrompt = PromptCalculator.adaptPromptToConversationAndModel(prompt, scope)
 
-  suspend fun createChatCompletion(request: ChatCompletionRequest): ChatCompletionResponse
+  val request =
+    CreateChatCompletionRequest(
+      stream = true,
+      user = prompt.configuration.user,
+      messages = messagesForRequestPrompt.messages,
+      n = prompt.configuration.numberOfPredictions,
+      temperature = prompt.configuration.temperature,
+      maxTokens = prompt.configuration.minResponseTokens,
```

Comment:
@raulraja I don't know if this is a typo or if it is managed in this way. I see confusing the name that is put in the promptConfiguration and then it is set in parameters with the name "maxTokens", etc... 

This happens a few more times in the code...

Reply:
The name in `CreateChatCompletionRequest` which is the generated model is `maxTokens`, I'll rename the config one to match it.